app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :foods
+  has_many :recipes
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- Hi, Your App is OK. Now it is time to clean and **remove the unused comment**.  Because your code needs to be clean